bundles/nightly/pom.xml

```diff
@@ -123,7 +123,30 @@
                             </reportsDirectories>
                             <outputDirectory>${project.build.directory}/jpa-test-reports/${project.build.testReports.subdirectory}</outputDirectory>
                             <outputName>${project.build.testReports.summaryFile}</outputName>
-                            <title>EclipseLink JPA Test - Test Summary (more details on ${env.BUILD_URL} )</title>
+                            <customBundle>${project.basedir}/src/site/custom/surefire-report.properties</customBundle>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>prepare-nightly-build-dir-test-reports-resources</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <copy todir="${project.build.directory}${nightlyDir}/Eclipse/css/">
+                                    <fileset dir="../../foundation/eclipselink.core.test/target/${project.build.testReports.subdirectory}/css/"/>
```

Comment:
can the content be taken from test-assembly (or some other build-time artifact) to avoid depending on some, possibly random, paths?

Reply:
I tried find some solution with origin XML reports (see _target/failsafe-reports_), but there is no any single overall XML report. Just only number of XML reports one per each test suite. I didn't find any tool which should merge them (surefire + failsafe) except `maven-surefire-report-plugin`. But `maven-surefire-report-plugin` can produce only HTML aggregated output, but not XML.